src/Chorus/UI/Settings/SendReceiveSettings.cs

```diff
@@ -20,10 +24,25 @@ public SendReceiveSettings()
 			InitializeComponent();
 		}
 
+		/// <summary>
+		/// Make sure that the html content is displayed properly on both linux and windows by using Navigate on the chorusHubSetup control after the dialog is shown
+		/// </summary>
+		private void SendReceiveSettingsShown(object sender, EventArgs e)
+		{
+			System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(SendReceiveSettings));
+			var hubSetupText = LocalizationManager.GetString(@"Chorus_ChorusHubSetupInstructionsHtml", resources.GetString(@"ChorusHubSetupInstructionsHTML"),
+				@"Instructions shown before first Send/Receive. Please keep HTML tags as-is to preserve formatting, and use HTML escapes");
```

Comment:
(I didn't entirely understand why the "Please keep HTML tags..." line was moved into here from the resource file, but okay.)